ivy/functional/frontends/torch/indexing_slicing_joining_mutating_ops.py

```diff
@@ -148,3 +148,13 @@ def dstack(tensors, *, out=None):
 @to_ivy_arrays_and_back
 def take_along_dim(input, indices, dim, *, out=None):
     return ivy.take_along_axis(input, indices, dim, out=out)
+
+
+@to_ivy_arrays_and_back
+def where(condition, input=None, other=None, *, out=None):
+    if input is None and other is None:
+        return ivy.nonzero(condition)
+    elif input is not None and other is not None:
+        return ivy.where(condition, input, other, out=out)

```

Comment:
`ivy.where` is called the ivy torch backend implementation [here](https://github.com/unifyai/ivy/blob/abb765da1cb2b7b14de03220e06061c5efd7d14b/ivy/functional/backends/torch/searching.py#L86). Currently x1 and x2 only accept tensor types, but according to the torch.where documentation, it should also accept scalars. I will update this backend function now and you can pull the fix from master. Your frontend implementation should work then.

Reply:
@LucaLumetti I have update the `where` torch backend, please pull from master and see if your PR passes the checks 😊